vbar_window.vala

```diff
@@ -21,6 +21,14 @@ public class VbarWindow : Gtk.ApplicationWindow {
       this.application.quit();
       return;
     }
+    
+    // Enable transparency
```

Comment:
Could you remove the comment and extract a function called `enable_transparency` or something like that?